REDCapUITweaker.php

```diff
@@ -1203,6 +1275,54 @@ function provideSQLDescriptive( $listFields )
 
 
 
+	// Output the current user role name
+
+	function provideUserRoleName()
+	{
+		if ( !defined( 'USERID' ) || USERID == '' )
+		{
+			return;
+		}
+
+		$userRights = $this->getUser()->getRights();
+		if ( !isset( $userRights ) ||
+		     !isset( $userRights['role_name'] ) || $userRights['role_name'] == '' )
+		{
+			return;
+		}
+
+		$roleName = $this->escapeHTML( $userRights['role_name'] );
+
+?>
+<script type="text/javascript">
+  $(function()
+  {
+    if ( $('#user-role-name').length > 0 )
+    {
+      return
+    }
+    var vRoleName = $('<div style="' + $('#username-reference').parent().attr('style') + '">Role:' +
+                     ' <span id="user-role-name" style="' + $('#username-reference').attr('style') +
+                      '"><?php echo $roleName; ?></span></div>')
+    vRoleName.css('margin-top','-4px').css('margin-left','17px')
```

Comment:
It's good idea to look at parent style then change the vRolename css but it's not line up.  Can you change it precisely with the line 'Logged in .... '?  Thanks!
vRoleName.css({margin-top:'-4px',margin-left: '14px'});

Reply:
I have amended this to improve the alignment.